main/cli.c

```diff
@@ -807,13 +807,17 @@ static char *handle_logger_mute(struct ast_cli_entry *e, int cmd, struct ast_cli
 
 static char *handle_refresh(struct ast_cli_entry *e, int cmd, struct ast_cli_args *a)
 {
+	int res;
 	/* "module refresh <mod>" */
 	switch (cmd) {
 	case CLI_INIT:
 		e->command = "module refresh";
 		e->usage =
-			"Usage: module refresh <module name>\n"
-			"       Unloads and loads the specified module into Asterisk.\n";
+			"Usage: module refresh <module name> [automatically]\n"
```

Comment:
How about "recursively" instead of "automatically"?
Also, can you make the tab-autocomplete work after you've entered the module name?